Java/Java.sublime-syntax

```diff
@@ -227,7 +338,15 @@ contexts:
                 - match: "}"
                   pop: true
                 - include: class-body
+
   keywords:
+    - match: '::'
+      scope: keyword.control.method-reference.java
+      push:
+        - match: '{{id}}'
```

Comment:
Yeah, now that you confirmed this is for static references to methods, I believe that would be the correct scope.